bootleg/run.py

```diff
@@ -99,7 +98,9 @@ def setup(config, run_config_path=None):
         config: config
         run_config_path: path for original run config
     """
-    # torch.multiprocessing.set_sharing_strategy("file_system")
+    if "mp_sharing_strategy" in config.data_config:
```

Comment:
One more comment on this, so the mp_sharing_strategy will always be in the config. It'll just have the value of "". The better way to do it is the set the default sharing strategy to be None in `bootleg_args.py` and then check in this statement if the strategy is not None.

Reply:
done